functions/Find-DbaDbUnusedIndex.ps1

```diff
@@ -72,11 +96,40 @@ function Find-DbaDbUnusedIndex {
         [switch]$IgnoreUptime,
         [Parameter(ValueFromPipeline)]
         [Microsoft.SqlServer.Management.Smo.Database[]]$InputObject,
-        [switch]$EnableException
+        [switch]$EnableException,
```

Comment:
Please place all new parameters prior to the `$InputObject` parameter in this param block.

Reply:
ok, will do